.github/workflows/quality-checks.yml

```diff
@@ -0,0 +1,15 @@
+name: Smart Events UI Quality Checks
+on: [pull_request, push]
+jobs:
+  run-all-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+          cache: "npm"
+      - run: npm install
+      - run: npm run format:check
+      - run: npm run lint
+      - run: npm run test
```

Comment:
is one of these commands already covering type checking?

Reply:
I tend to say no, beacuse of https://issues.redhat.com/browse/MGDOBR-478, may I ask you to add such a requirement into linked jira?

Reply:
Ok, I updated the jira. Thank you